commerce/src/inline-price.js

```diff
@@ -14,11 +20,78 @@ const DISPLAY_ALL_TAX_COUNTRIES = [GeoMap.uk, GeoMap.au, GeoMap.fr, GeoMap.at, G
     GeoMap.in_en, GeoMap.in_hi, GeoMap.jp, GeoMap.my_en, GeoMap.my_ms, GeoMap.nz, GeoMap.th_en, GeoMap.th_th];
 // countries where tax is displayed for some segments only by default
 const DISPLAY_TAX_MAP = {
-    'INDIVIDUAL_COM': [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.sa_ar, GeoMap.sa_en, GeoMap.za, GeoMap.sg, GeoMap.kr], // individual
-    'TEAM_COM': [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.za, GeoMap.co, GeoMap.kr], // business
-    'INDIVIDUAL_EDU': [GeoMap.lt, GeoMap.lv, GeoMap.sa_en, GeoMap.sea], // student
-    'TEAM_EDU': [GeoMap.sea, GeoMap.kr], // school and uni
+    [INDIVIDUAL]: [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.sa_ar, GeoMap.sa_en, GeoMap.za, GeoMap.sg, GeoMap.kr],
+    [BUSINESS]: [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.za, GeoMap.co, GeoMap.kr],
+    [STUDENT]: [GeoMap.lt, GeoMap.lv, GeoMap.sa_en, GeoMap.sea],
+    [UNIVERSITY]: [GeoMap.sea, GeoMap.kr],
+};
+
+// For most countries where tax label is displayed the tax is included for Individuals and Students
+// and excluded for Business and Universities. This is the map of TaxExclusive values for other countries
+const TAX_EXCLUDED_MAP = {
+    [GeoMap.za]: [false, false, Defaults.forceTaxExclusive, Defaults.forceTaxExclusive],
+    [GeoMap.at]: [false, false, false, true],
+    [GeoMap.ng]: [false, false, Defaults.forceTaxExclusive, Defaults.forceTaxExclusive],
+    [GeoMap.au]: [false, false, false, false],
+    [GeoMap.jp]: [false, false, false, false],
+    [GeoMap.nz]: [false, false, false, false],
+    [GeoMap.th_en]: [false, false, false, false],
+    [GeoMap.th_th]: [false, false, false, false],
 };
+const TAX_EXCLUDED_MAP_INDEX = [INDIVIDUAL, BUSINESS, STUDENT, UNIVERSITY];
+const TAX_EXCLUDED_MAP_OTHER = {
+    [INDIVIDUAL]: false,
+    [BUSINESS]: true,
+    [STUDENT]: false,
+    [UNIVERSITY]: true,
+};
+
+const resolveTaxExclusive = (country, language, customerSegment, marketSegment) => {
+    const locale = `${country}_${language}`;
+    const segment = `${customerSegment}_${marketSegment}`;
+    const val = TAX_EXCLUDED_MAP[locale];
+    if (val) {
+        const index = TAX_EXCLUDED_MAP_INDEX.indexOf(segment);
+        return val[index];
+    }
+
+    return TAX_EXCLUDED_MAP_OTHER[segment];
+}
+
+/**
+ * Resolves default value of displayTax property, based on provided geo info and segments.
```

Comment:
can you add jsdoc for all parameters.
we particularly care for the documentation of the commerce lib.

Reply:
jsdoc added